integration/conftest.py

```diff
@@ -544,17 +545,9 @@ def chutney(argv):
         return proto.done
 
     # now, as per Chutney's README, we have to create the network
-    # ./chutney configure networks/basic
-    # ./chutney start networks/basic
     pytest_twisted.blockon(chutney(("configure", basic_network)))
-    pytest_twisted.blockon(chutney(("start", basic_network)))
-
-    # print some useful stuff
-    try:
-        pytest_twisted.blockon(chutney(("status", basic_network)))
-    except ProcessTerminated:
-        print("Chutney.TorNet status failed (continuing)")
 
+    # ensure we will tear down the network right before we start it
```

Comment:
Hm.  I don't understand this.  Before, we started the network and then set up the finalizer.  Now we set up the finalizer and then start the network.  I don't see how either order would lead to tearing down the network before we started it.

Reply:
I guess that's worded poorly: I mean _arrange_ for it to be torn down first.